### PR TITLE
Add unit test to VolumeCreate

### PR DESCRIPTION
**Pull request by vdemeester**
Time for `VolumeCreate` 🐇.

🐸

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: client/volume_create_test.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+				Body:       ioutil.NopCloser(bytes.NewReader(content)),
+			}, nil
+		}),
+	}
+
+	volume, err := client.VolumeCreate(types.VolumeCreateRequest{
+		Name:   "myvolume",
+		Driver: "mydriver",
+		DriverOpts: map[string]string{
+			"opt-key": "opt-value",
```
Comment (calavera) [inline client/volume_create_test.go]: we're not checking this options, is it correct?

Comment (vdemeester) [inline client/volume_create_test.go]: Yep, but we probably should :stuck_out_tongue_closed_eyes:

**Comment by duglin** [comment]
LGTM

**Comment by calavera** [comment]
LGTM

**Pull request merged**